### PR TITLE
Update

### PR DESCRIPTION
**Pull request by franktcurran**
This branch is about updating ember-cli to 3.2.0. Most recommended updated code styles applied (not closure actions), and updated many dependencies.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,35 +1,47 @@
...
+    - EMBER_TRY_SCENARIO=ember-lts-2.18
+    - EMBER_TRY_SCENARIO=ember-release
+    - EMBER_TRY_SCENARIO=ember-beta
+    - EMBER_TRY_SCENARIO=ember-canary
+    - EMBER_TRY_SCENARIO=ember-default
 
 matrix:
   fast_finish: true
   allow_failures:
-    - env: EMBER_TRY_SCENARIO=ember-beta
```
Comment (rynam0) [inline .travis.yml]: I would put this back here as its probably ok if beta fails.

File: .travis.yml [hunk 2]
```diff
@@ -1,35 +1,47 @@
...
 before_install:
   - npm config set spin false
-  - npm install -g bower
-  - bower --version
-  - npm install phantomjs-prebuilt
-  - phantomjs --version
-
-install:
-  - npm install
-  - bower install
```
Comment (rynam0) [inline .travis.yml]: I think you'll need the bower install to go back in here too.

File: blueprints/ember-cli-vtkui/index.js [hunk 3]
```diff
@@ -4,15 +4,15 @@ module.exports = {
   beforeInstall: function(options) {
```
Comment (rynam0) [inline blueprints/ember-cli-vtkui/index.js]: The options arg could probably be commented as it isn't used.

File: .travis.yml (hunk 1 above)
Comment (franktcurran) [reply .travis.yml]: Ah, things lost when I did the ember-cli-update, thanks Ryan!

**Pull request merged**